kroxylicious-systemtests/pom.xml

```diff
@@ -203,6 +203,12 @@
             <groupId>io.skodjob</groupId>
             <artifactId>test-frame-log-collector</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.kroxylicious</groupId>
+            <artifactId>kroxylicious-operator-test-support</artifactId>
+            <version>${project.version}</version>
```

Comment:
Push the version up to the parent, in dependencyManagement.

Reply:
Its not the only usage of `${project.version}` in the system test dependencies and I'm not sure moving to dependency management improves things. 

What I'd really like to do is build a bom for the operator and then pull that in here.